src/install/aws/ui-auth.md

```diff
@@ -0,0 +1,27 @@
+### UI Authentication
+
+To provide authentication  for UI  [Amazon Cognito](https://aws.amazon.com/cognito/) utilized.
+As a part of MDAI stack installation, the following aws resources will be created automatically:
+
+[User pool](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools.html) `mdai-user-pool`
+
+[App client](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-client-apps.html) `mdai-app-client`
+
+#### Pre-build configuration
+
+The following environment variables need to be defined in `values/aws.env` during the pre-build configuration step:
+
+`MDAI_UI_HOSTNAME`
+This is a hostname for the UI endpoint, which is going to be used as a main UI hostname. 
+It's supposed to be a DNS CNAME record, pointing to DNS hostname, generated by AWS for the Load Blancer `mdai-console` endpoint. 
+
+`MDAI_UI_USER_POOL_DOMAIN`
+This variable contains a prefix for [User pool domain](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-assign-domain.html).
+[Amazon Cognito domain](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-assign-domain-prefix.html)
+is used with this installation, so this variable represents a prefix, for the domain.
+This domain name will become a part of the auth redirecting URL.
+
+#### User(s) creation
+The only manual step required is a user(s) creation.
```

Comment:
Can we link to docs here? The less we write out steps for a service/tool we don't support the better. If they change UI, we'd have to maintain